src/stashbox.extension.wcf/PerServiceInstanceLifetime.cs

```diff
@@ -8,14 +8,12 @@ namespace Stashbox.Extension.Wcf
 {
     public sealed class PerServiceInstanceLifetime : LifetimeBase
     {
+        private volatile Expression _expression;
```

Comment:
Thanks for the detailed explanation and the link to further reading and thanks again for taking the time to update the extension.